lib/client.c

```diff
@@ -1527,6 +1527,7 @@ sf_put_get_response_deallocate(SF_PUT_GET_RESPONSE *put_get_response) {
 
     snowflake_cJSON_Delete((cJSON *) put_get_response->src_list);
     snowflake_cJSON_Delete((cJSON *) put_get_response->enc_mat_get);
+    snowflake_cJSON_Delete((cJSON*)put_get_response->presigned_urls);
```

Comment:
nitpicking: can we format the code as above

Reply:
done